controllers/automation/packagedeployment_controller.go

```diff
@@ -126,7 +128,8 @@ func (r *PackageDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	//  - Create new PackageRevisions for new matches
 	//  - Verify the version for existing matches
 	//
-	// This code ONLY creates new PackageRevisions for new matches
+	// This code ONLY creates new PackageRevisions for new matches and performs updates
```

Comment:
s/ONLY //

Maybe also say. "It does not yet do deletions or pruning of PackageRevisions" to make it clear what is missing.